components/picker/index.css

```diff
@@ -10,409 +10,535 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+       var(--mod-picker-focus-indicator-thickness, var(--spectrum-picker-focus-indicator-thickness)) +
+       var(--mod-picker-border-size, var(--spectrum-picker-border-size))) *
+      -1
+    );
+    border-style: solid;
+    border-width: var(--mod-picker-focus-indicator-thickness, var(--spectrum-picker-focus-indicator-thickness));
+    border-color: transparent;
+    border-radius: calc(
+      var(--mod-picker-border-radius, var(--spectrum-picker-border-radius)) +
+      var(--mod-picker-focus-indicator-gap, var(--spectrum-picker-focus-indicator-gap)) +
+      var(--mod-picker-border-width, var(--spectrum-picker-border-width))
+    );
+  }
 
-.spectrum-Picker-popover--quiet {
-  margin-inline-start: calc(
-    -1 * calc(var(--spectrum-picker-m-quiet-texticon-popover-offset-x) +
-          var(--spectrum-popover-border-size))
-  );
-}
+  &:focus-ring,
+  &:focus-visible {
+    /* Remove native outline */
+    outline: none;
 
-.spectrum-Picker {
-  color: var(--spectrum-picker-m-texticon-text-color);
-  background-color: var(--spectrum-picker-m-texticon-background-color);
-  border-color: var(--spectrum-picker-m-texticon-border-color);
+    /* Focus ring */
+    &::after {
+      border-color: var(--highcontrast-picker-focus-indicator-color, var(--mod-picker-focus-indicator-color, var(--spectrum-picker-focus-indicator-color)));
+    }
+  }
 
   &:hover {
-    color: var(--spectrum-picker-m-texticon-text-color-hover);
-    background-color: var(--spectrum-picker-m-texticon-background-color-hover);
-    border-color: var(--spectrum-picker-m-texticon-border-color-hover);
+    color: var(--highcontrast-picker-font-color-default, var(--mod-picker-font-color-hover, (--spectrum-picker-font-color-hover)));
+    background-color: var(--highcontrast-picker-background-color-default, var(--mod-picker-background-color-hover, var(--spectrum-picker-background-color-hover)));
+    border-color: var(--highcontrast-picker-border-color-default, var(--mod-picker-border-color-hover, var(--spectrum-picker-border-color-hover)));
 
     .spectrum-Picker-menuIcon {
-      color: var(--spectrum-picker-m-texticon-icon-color-hover);
+      color: var(--highcontrast-picker-icon-color-hover, var(--mod-picker-icon-color-hover, var(--spectrum-picker-icon-color-hover)));
     }
   }
 
-  &:active,
-  &.is-open {
-    background-color: var(--spectrum-picker-m-texticon-background-color-down);
-    border-color: var(--spectrum-picker-m-texticon-border-color-down);
+  &:active {
+    background-color: var(--highcontrast-picker-background-active, var(--mod-picker-background-color-active, var(--spectrum-picker-background-color-active)));
+    border-color: var(--highcontrast-picker-border-color-active, var(--mod-picker-border-active, var(--spectrum-picker-border-color-active)));
+
+    &::after {
+      border-color: transparent;
+    }
 
     &.is-placeholder {
       .spectrum-Picker-label {
```

Comment:
This nesting does not work because

`<span class="spectrum-Picker-label is-placeholder">`

.spectrum-Picker-label and is-placeholder are the same element, not nested.

Reply:
Thanks, updated!